components/core/JsInterop/interop.ts

```diff
@@ -672,4 +672,21 @@ export function setDomAttribute(element, attributes) {
   for (var key in attributes) {
     (dom as HTMLElement).setAttribute(key, attributes[key]);
   }
-}
\ No newline at end of file
+}
+
+export function getSelectionStart(element) {
+    if (element) {
+        let dom = getDom(element);
+        return dom.selectionStart;
```

Comment:
I see. This if check seems kinda pointless, right? Just saw it in `removePreventEnterOnOverlayVisible` and thought that this check would come in handy.
I have removed the check.

Reply:
Nope, I mean that we can use `getDom` method to get the `selectionStart` property.

Reply:
Then I admit that I don't understand how `getDom` can be used to retrieve a value of a property. If '"selectionStart"' (or any other string) is passed to `getDom` then `document.querySelector` will be invoked and it will try to find an element not a property, will it not?
In *interop.ts* I haven't found any example which could help me.
Therefore, could you please share a code snippet how `getDom` could be called to retrieve a property - I will try to run it.
Sorry for my ignorance 😞

Reply:
We have used `getDomInfo` to get properties of specific dom.

```
 HtmlElement element = await JsInvokeAsync<HtmlElement>(JSInteropConstants.GetDomInfo,Ref);
```

Reply:
oh, so I needed to add *selectionStart* to `getDomInfo`. Did it.
I had a dilemma about what default value to use for `selectionStart`. Usually, it's **-1**, but in the end I decided to opt for 0 because it's much easier to use.